src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -165,6 +165,8 @@ private ConfigurationLoader(final PropertyResolver overrideProps,
     /**
      * Creates mapping between local resources and dtd ids.
      * @return map between local resources and dtd ids.
+     * @noinspection MethodOnlyUsedFromInnerClass method must be static,
+     *     and inner class isn't static
```

Comment:
inspections do not support definition of reason of suppress, in tag `@noinspection` there should be only list of inspection names.

`method must be static, and inner class isn't static` should be moved to javadoc description. Method is private, it will not be visible in HTML.

Reply:
Done.

I just made it a new sentence. Let me know if you want it documented another way.